deploy/cr.yaml

```diff
@@ -743,3 +743,10 @@ spec:
 #        storageName: s3-us-west
 #        compressionType: gzip
 #        compressionLevel: 6
+  logcollector:
+    enabled: true
+    image: perconalab/percona-xtradb-cluster-operator:main-logcollector
```

Comment:
we need to fix this image, is this temporary?

Reply:
Until the new images for Fluentbit are ready, I think we should keep this as is. We can change it with a new PR once these new images are ready.